README.md

```diff
@@ -47,12 +47,15 @@ Usage: ovpnmcgen.rb generate [options] <user> <device>
     -c, --config FILE    Specify path to config file. [Default: .ovpnmcgen.rb.yml]
     --cafile FILE        Path to OpenVPN CA file. (Required)
     --tafile FILE        Path to TLS-Auth Key file.
+    --cert FILE          Path to Cert file.
+    --key FILE           Path to Private Key file.
     --host HOSTNAME      Hostname of OpenVPN server. (Required)
     --proto PROTO        OpenVPN server protocol. [Default: udp]
     -p, --port PORT      OpenVPN server port. [Default: 1194]
-    --p12file FILE       Path to user PKCS#12 file. (Required)
+    --p12file FILE       Path to user PKCS#12 file.
     --p12pass PASSWORD   Password to unlock PKCS#12 file.
     --[no-]vod           Enable or Disable VPN-On-Demand. [Default: Enabled]
+    --v12compat          Enable OpenVPN Connect 1.2.x compatibility. [Default: Disabled]
```

Comment:
It occurs to me that the `--v12compat` behavior might make more sense as the default and the old behavior could be stashed behind a `--v11compat` flag.

Reply:
It’ll be a breaking change that I’m happy to make when the major version number gets bumped higher later. For now, I’m focusing on getting something working reliably first.

Reply:
consider that soon p12 payloads bundled into mobileconfig files will be allowed again. It is just a transient moment where the App can't support them, but they'll be back shortly.